src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -1150,9 +1156,27 @@ def __call__(self, *args, **kwds) -> Any:
         request.path = convert_path_to_se_path(self.path)
         request.command = self.command
         request.wait = True
+        if "FileName" in kwds:
+            kwds.update(
+                dict(
+                    FileName=os.path.basename(kwds["FileName"])
+                    if bool(self.service.remote_file_handler)
+                    else kwds["FileName"]
+                )
+            )
         _convert_value_to_variant(kwds, request.args)
+        file_purpose = None
+        if hasattr(self.create_instance(), "get_attr"):
```

Comment:
You keep calling `create_instance` here. This is triggering remote calls that are actually creating command instances in the server.

Reply:
Yes, right. Therefore most of the tests are failing.